extension/ezoe/modules/ezoe/upload.php

```diff
@@ -122,25 +122,32 @@
 
         $uploadVersion = $uploadedOk['contentobject']->currentVersion();
         $newObjectID = (int)$uploadedOk['contentobject']->attribute( 'id' );
+        $imageAltText = '';
 
         foreach ( $uploadVersion->dataMap() as $key => $attr )
         {
             //post pattern: ContentObjectAttribute_attribute-identifier
             $base = 'ContentObjectAttribute_'. $key;
-            $postVar = trim( $http->postVariable( $base, '' ) );
-            if ( $postVar !== '' )
+            if( $http->hasPostVariable( $base ) )
             {
+                $postVar = trim( $http->postVariable( $base, '' ) );
                 switch ( $attr->attribute( 'data_type_string' ) )
                 {
                     case 'ezstring':
-                        $classAttr = $attr->attribute( 'contentclass_attribute' );
-                        $dataType = $classAttr->attribute( 'data_type' );
-                        if ( $dataType->validateStringHTTPInput( $postVar, $attr, $classAttr ) !== eZInputValidator::STATE_ACCEPTED )
+                        // Not very clean, it prevents overwriting the object name
```

Comment:
Philipp might be better at explaining but I believe my initial changes incorporated something from a different pull request, so this is resetting that I think?